foundcareers/redux-entity
# FC-2517 - Added selector and selector tests, updated factory tests

**QA**

1. Read through the following files:
- selector.test.js
- selector.js
2. Check if the tests run. 
`npm test` or `yarn test`
